docker-compose-oracle.yml

```diff
@@ -3,12 +3,12 @@ version: '3'
 services:
   dbzui-zookeeper:
     container_name: dbzui-zookeeper
-    image: debezium/zookeeper:1.8
+    image: debezium/zookeeper:2.1
```

Comment:
```suggestion
    image: quay.io/debezium/zookeeper:2.1
```

Reply:
@rk3rn3r @Naros I have the same error with the `quay.io` images.